bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -513,7 +513,10 @@ static function (Task $task) {
             }
 
             if ($currentLayoutId === null && count($validLayouts) > 0) {
-                $currentLayoutId = reset($validLayouts)->getId();
+                $currentLayout = $validLayouts['0'] ?? reset($validLayouts);
```

Comment:
If we change the ids we then have to check for master here right?
```suggestion
                $currentLayout = $validLayouts['master'] ?? reset($validLayouts);
```